src/atomic.dex.app.cpp

```diff
@@ -908,14 +909,19 @@ namespace atomic_dex
                 get_mm2().apply_erc_fees(receive_ticker.toStdString(), erc_fees);
             }
 
-            auto tx_fee_value     = QString::fromStdString(get_formated_float(tx_fee_f));
-            auto final_balance_f  = t_float_50(amount.toStdString()) - (trade_fee_f + tx_fee_f);
-            std::string final_balance = amount.toStdString();
-            if (final_balance_f > 0)
+            auto        tx_fee_value    = QString::fromStdString(get_formated_float(tx_fee_f));
+            auto        final_balance_f = t_float_50(amount.toStdString()) - (trade_fee_f + tx_fee_f);
+            std::string final_balance   = amount.toStdString();
+            // spdlog::info("final_balance_f: {}", get_formated_float(final_balance_f));
+            if (final_balance_f.convert_to<float>() > 0.0)
```

Comment:
tagging this "hackaround" for perf-improvement in the next iteration - cc: @Milerius